### ISSUE TITLE
No speech on Mac OS

### ISSUE THREAD

**Issue by pitermach** [lightsoutgames/godot-tts#4]
While trying to help someone test out accessibility for their upcoming Godot game, I discovered the TTS plugin appears to now be broken on Mac OS. When starting said game, I got no speech and the following output in the terminal (not sure if this is a GodotTTS message or something put in by the developer):

> TTS isn't set up, can't speak text. Insall an appropriate speech synthesis tool for your OS. Compatible tools are listed here https://github.com/lightsoutgames/godot-tts

Trying to run Onslaught gave me a similar result of no speech, though also no output like this if ran from the terminal.

System specs, MacBook Air M1, confirmed broken on both Mac OS 11.2 and 11.3 public beta.

**Comment by ndarilek** [comment]
Are both these failures on M1 processors?


I'll test on my Intel Mac later this week and see whether it fails there 
as well.

**Comment by pitermach** [comment]
Yes. One of them was tested by me on 11.3, the 11.2 test was done by a friend on identical hardware.

**Comment by Cheeseness** [comment]
> (not sure if this is a GodotTTS message or something put in by the developer):

If it's helpful to know, that's something I output when `is_valid_instance()` on the Godot-TTS node returns false.

**Comment by ndarilek** [comment]
I'm not able to reproduce lack of speech on my Intel mac updated to the latest macOS build. So I'm going to need those of you with M1s to help debug this.

First step would be determining if [tts-rs(https://github.com/ndarilek/tts-rs) runs on the M1. Once a Rust toolchain is set up, running:

`cargo run --example hello_world`

should start speaking once everything builds. Given that I assume Godot is using the X86 builds, it'd be helpful if you could build with the X86 Rust toolchain. Something like:

`cargo run --example hello_world --target x86_64-apple-darwin`

should do that once the relevant target is installed.

Another thing that could be going on is that Godot isn't loading the X86 library in the addon. An easy way to check this would be changing:

```
OSX.64="res://addons/godot-tts/target/debug/libgodot_tts.dylib"
```

in godot-tts.gdnlib to:

```
OSX="res://addons/godot-tts/target/debug/libgodot_tts.dylib"
```

Not sure what else to try. If someone wants to give me an SSHable account on an M1 box that they don't mind speaking from time to time, I can try connecting and getting things to build. I know the TTS crate has issues building on ARM, but I'm assuming that's not the problem here since I'm guessing you're emulating. Or, at least, if you're not then you should be since there is no ARM build of this addon. :)

**Comment by Cheeseness** [comment]
Thanks for the tips!

> Given that I assume Godot is using the X86 builds

Looks like Godot's doing 64 bit intel builds for Mac for my game at least.

`Mach-O 64-bit x86_64 executable, flags:<NOUNDEFS|DYLDLINK|TWOLEVEL|WEAK_DEFINES|BINDS_TO_WEAK|PIE|HAS_TLV_DESCRIPTORS>`

**Comment by ndarilek** [comment]
Sorry, meant X86 vs. Arm.
